### ISSUE TITLE
QUEST and staircase files

### ISSUE THREAD

**Issue by ASmitho** [psychopy/psychojs#48]
Hi, 
I was looking through the files and cannot see any functions involving staircasing or QUEST thresholding. These are in psychopy and I was wondering if they were going to be in psychojs. Thanks.

**Comment by peircej** [comment]
I don't think this will be a feature we'll add urgently but, yes, at some point it would be great to support staircases too

**Comment by Ashwani270292** [comment]
Hey, just checking, when can we expect Staircase feature?
